### PR TITLE
Use correct cellcenterdepth

### PR DESCRIPTION
**Pull request by dr-robertk**
This replaces the grid.cellCenterDepth method with the correct DUNE interface usage for the same computation.

### CODE REVIEW

**Comment by dr-robertk** [comment]
@andlaus: I have rebase on top of your PR. Making a PR for your local branch did not work.

**Comment by dr-robertk** [comment]
jenkins build this please

**Comment by andlaus** [comment]
good idea to do this generically. why did you close this?

**Comment by dr-robertk** [comment]
Accident.

File: applications/ebos/eclproblem.hh [hunk 1]
```diff
@@ -818,13 +833,14 @@ private:
         auto elemIt = gridView.template begin</*codim=*/0>();
         const auto& elemEndIt = gridView.template end</*codim=*/0>();
         for (; elemIt != elemEndIt; ++elemIt) {
+            const Element& element = *elemIt;
 #if DUNE_VERSION_NEWER(DUNE_COMMON, 2,4)
-            unsigned elemIdx = elemMapper.index(*elemIt);
+            const unsigned int elemIdx = elemMapper.index(element);
```
Comment (andlaus) [inline applications/ebos/eclproblem.hh]: I generally dislike declaring local variables and call-by-value method arguments as `const`, IMO it is just a distraction. Mind removing it? (I won't strictly require it.)

**Comment by andlaus** [comment]
ah, travis ;)

**Comment by dr-robertk** [comment]
@andlaus I don't mind

File: applications/ebos/eclproblem.hh (hunk 1 above)
Comment (andlaus) [inline applications/ebos/eclproblem.hh]: well, on a personal "annoyingness scale" from 0 to 100 I would put it at about 8. (that's probably about the same score as curly brackets for each 'if' get, btw.) I just don't see any value in these qualifiers because they've never helped me to produce better code so far...

tl;dr: don't mind and do what you prefer!

**Comment by andlaus** [comment]
since the release branch has been created and travis is happy, there's no reason to hold this back. thanks.
